Docs/ROADMAP.md

```diff
@@ -0,0 +1,328 @@
+# Roadmap for Manafest: Arena 2
+
+This document is intended to provide context and a detailed description of goals for the development of the sequel to Manafest: Arena.
+
+## Recap of Manafest: Arena
+
+After a few months of development on the FPS_Framework repo, I decided it was time to make a video game using what I had done already as a starting point.
+With a three month timeline, Manafest: Arena was developed.
+
+### Initial build
+
+The goal was relatively simple: I wanted to produce an FPS rogue-like with elements similar to Slay The Spire https://store.steampowered.com/app/646570/Slay_the_Spire/ .
+The player choose one of three unique starting characters to begin their adventure. They are then presented a map consisting of a tree of encounters, and are given the choice of which encounter to start with on one of the leaves of this tree. Encounters included random Arena matches as well as rest sites where the player can choose between an upgrade for their character, or healing.
+
+A couple features were scrapped due to lack of time. One feature was a shop encounter that would allow the player to acquire new items for combat. Another feature was a press event encounter in which the player would respond to a multiple choice prompt and reward or punish the player based on their choice and random chance. Lastly, there was a sponsorship feature which would grant abilities or bonuses to the player when earned. Before work was started, all netcode was pulled from the code base.
```

Comment:
Might be better to put into a different document. Having the roadmap be as lean as possible makes it easier for others to scan and get into. Also just might get more easily lost in here if they are features that aren't making it into v1 anyway